cases/src/org/commcare/cases/ledger/instance/LedgerInstanceTreeElement.java

```diff
@@ -93,7 +93,7 @@ public void attachStringCache(Interner<String> stringCache) {
      */
     public LedgerChildElement getChild(String name, int multiplicity) {
         if (multiplicity == TreeReference.INDEX_TEMPLATE) {
-            return null;
+            return LedgerChildElement.TemplateElement(this);
```

Comment:
The fix; not sure if this has widespread implications or not.

Currently changes the code path in ```DataInstance.hasTemplatePathRec``` to detect a template path even when no ledgers have been loaded via ```currentNode.getChildrenWithName```, which in turn calls ```LedgerInstanceTreeElement.getLedgers```